### PR TITLE
Fixed items and formattings

### PR DESCRIPTION
**Pull request by thakyZ**
I fixed the formatting for most files but I haven't yet looked through all of them.
I have fixed the pistols showing up as perfectly generic items.
I have fixed the avali armor showing up as perfectly generic items.
(On my side only) I messed up and changed the ship's textures and files which are now fixed.
And I removed the end of line requirement in .gitattributes because I have now fixed the formatting of all files they should download normal now.
Fixed a crash with the Avali Laser Miner.

### CODE REVIEW

**Comment by thakyZ** [comment]
@Fevix Ok, I got the files fixed now, so do you now have any objection to this pull request?

File: ai/ai.config.patch [hunk 1]
```diff
@@ -1,36 +1,34 @@
...
+    "openSpeech" : {
+      "0" : [ {
+        "animation" : "unique",
+        "text" : "ERROR: ERROR: Info:'S.A.I.L.::run.innerLoop', 'Cnt#: ', 1198, 'Sum:ms:', ERROR: 'Min', 15, 'Max', 7199, 'Horse', 9899, 'Avg', 23, 'Last:::', '800b5', 'Var:avg,us: ERROR: ERROR: REBOOT REQUIRED!!!'",
+        "speedModifier" : 0.5
+      } ],
+      "1" : [ {
         "animation" : "talk",
-        "text" : "Here you can augment your body with tech cards and nano suits. ^orange;Tech cards ^green;will give you unique abilities whilst ^orange;nano suits ^green;will allow you to survive in otherwise deadly environments.",
+        "text" : "The thrusters and FTL drive are offline. I can repair the thrusters with the ships auto repair module but I'll need ^orange;core fragments^green; from the core of the planet below.",
```
Comment (thakyZ) [inline ai/ai.config.patch]: BTW @Fevix some files are saying I changed something when I really didn't (i.e. the two lines on this change)

Comment (Fevix) [inline ai/ai.config.patch]: Unfortunately, the pull request WILL change these if I merge.

Comment (Fevix) [inline ai/ai.config.patch]: Looking at the full thing, I see that it's calling this a "change" because of the lines inserted above it.

**Comment by Fevix** [comment]
You're added on as a collaborator, so you have the ability to merge these yourself now. 

I did this because I trust you to be responsible. 
I won't be inspecting your pull requests from here on, merge them whenever you feel they are complete.

**Pull request merged**